packages/components/pie-card-container/src/defs.ts

```diff
@@ -46,10 +46,10 @@ export interface CardContainerProps {
      */
     isDraggable: boolean;
 
-     /**
-     * What the interaction intent of the card should be such as anchor, button or regular div (default).
+    /**
+     * What HTML element the card should be such as a or button.
```

Comment:
Nit: for readability
```suggestion
     * What HTML element the card should be such as `a` or `button`.
```